src/utils.js

```diff
@@ -316,6 +329,31 @@ const configChanged = (prevLambda, lambda) => {
   const keys = ['description', 'runtime', 'roleArn', 'handler', 'memory', 'timeout', 'env', 'hash']
   const inputs = pick(keys, lambda)
   const prevInputs = pick(keys, prevLambda)
+
+  if (prevLambda.securityGroupIds && (!lambda.vpcConfig || !lambda.vpcConfig.SecurityGroupIds)) {
+    return true
+  }
+
+  if (prevLambda.subnetIds && (!lambda.vpcConfig || !lambda.vpcConfig.SubnetIds)) {
+    return true
+  }
+
+  if (!prevLambda.securityGroupIds && lambda.vpcConfig) {
+    return true
+  }
+
+  if (!prevLambda.subnetIds && lambda.vpcConfig) {
+    return true
+  }
+
+  if (not(equals(lambda.vpcConfig.SecurityGroupIds, prevLambda.securityGroupIds))) {
+    return true
+  }
+
+  if (not(equals(lambda.vpcConfig.SubnetIds, prevLambda.subnetIds))) {
+    return true
+  }
+
```

Comment:
These checks could definitely be more elegant in the future.

This is testing:
1. is the vpc config different
2. before: no vpcConfig, now: we do
3. before: had vpcConfig, now: we dont